ArduCopter/Parameters.cpp

```diff
@@ -1229,6 +1229,13 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     AP_GROUPINFO("TKOFF_RPM_MAX", 7, ParametersG2, takeoff_rpm_max, 0),
 #endif
 
+    // @Param: FS_EKF_FILT
+    // @DisplayName: EKF Failsafe filter cutoff
+    // @Description: EKF Failsafe filter cutoff frequency. EKF variances are filtered using this value to avoid spurious failsafes from transient high variances. A higher value means the failsafe is more likely to trigger.
+    // @Range: 0 10
+    // @User: Advanced
+    AP_GROUPINFO("FS_EKF_FILT", 8, ParametersG2, fs_ekf_filt_hz, EKF_LPF_CUTOFF),
```

Comment:
it's strange that this is in copter, why not vehicle or in EKF?

Reply:
The EKF failsafe checks are all currently done in copter where there is already poor man's filtering

Reply:
> The EKF failsafe checks are all currently done in copter where there is already poor man's filtering

Do you mean the EKF_CHECK_ITERATIONS_MAX thing? 
'Cause in that case I wonder if we should be removing that now if we're replacing the functionality with the low pass filter?

Reply:
I wanted to change as little as possible in the logic